src/views/ApartmentApp/index.js

```diff
@@ -74,60 +144,64 @@ export default class ApartApp extends Component {
         this.setState({ applicants });
       }
     }
+    this.setState({ updating: false });
   };
 
   /**
-   * Loads the user's profile info only once (at start)
+   * Callback for apartment application save button
    */
-  async loadProfile() {
-    this.setState({ loading: true });
-    try {
-      const profile = await user.getProfileInfo();
-      this.setState({ userProfile: profile });
-      this.checkPersonType(profile);
-      if (this.state.isStu) {
-        let applicants = this.state.applicants;
-        applicants.push(profile);
-        this.setState({ applicants });
-      }
-      this.setState({ loading: false });
-    } catch (error) {
-      // Do Nothing
-    }
-  }
-
-  checkPersonType(profile) {
-    let personType = String(profile.PersonType);
-    this.setState({ isStu: personType.includes('stu') });
-    this.setState({ isFac: personType.includes('fac') });
-    this.setState({ isAlu: personType.includes('alu') });
-  }
+  handleSaveButtonClick = () => {
+    // The method is separated from callback because the housing API service must be handled inside an async method
+    let debugMessage = 'DEBUG: Save button was clicked';
+    console.log(debugMessage);
```

Comment:
Do you need this concole log?

Reply:
*console

Reply:
oh, the debug message itself is just for debugging, isn't it?

Reply:
I added some comments to clarify which things are for debugging and/or placeholder.
They will be highlighted if you are using the Better Comments extension in VSCode